src/natcap/invest/forest_carbon_edge_effect.py

```diff
@@ -804,59 +808,60 @@ def _calculate_tropical_forest_edge_carbon_map(
             row_coords[valid_edge_distance_mask].ravel(),
             col_coords[valid_edge_distance_mask].ravel()))
         # note, the 'n_jobs' parameter was introduced in SciPy 0.16.0
-        distances, indexes = kd_tree.query(
+        # for each forest point x, for each of its k nearest neighbors
+        distances, indexes = kd_tree.query( # distances: (x, k) indexes: (x, k)
```

Comment:
Ugh, sorry about this because it's so nitpicky, but inline comments need two spaces before the # don't they? If this is my only comment then no need to change this.

Reply:
Good catch! I was bending the rules to make it fit in 80 characters haha. I'll move it to its own line.